docs/reactive-nonblocking.qmd

```diff
@@ -0,0 +1,316 @@
+---
+title: Non-blocking operations
+---
+
+Sometimes in a Shiny app, you need to perform a long-running operation, like loading a large dataset or doing an expensive computation.
+If you do this in a reactive context, it will block the rest of the application from running until the operation completes.
+This can be frustrating for users, who may think that the app has crashed.
+
+Worse, if you have multiple users, then one user's long-running operation will block the other users' apps from running as well.
+These other users will not even be aware that their slow performance is due to another user's actions.
+
+In this article, we'll look at how to make your Shiny apps more responsive by using non-blocking operations.
+We'll also go out of our way to explain why the usual Python async techniques don't work the way you expect in Shiny.
```

Comment:
I'm not sure this is necessary. If you feel like it is, perhaps there's a less negative take on it?

Reply:
What started this whole thing was Shiny for Python users asking why using the async keyword didn't seem to make things concurrent. I do think it's quite surprising that we're a web framework that has async, yet it doesn't do the thing you expect async to do.

I'll tweak the wording